evap/static/css/evap.css

```diff
@@ -474,3 +474,7 @@ li.menu-indent {
 .table-striped tbody tr.sortable-ghost {
     background: #E5E5E5;
 }
+
+select {
+    width: 100%;
+}
```

Comment:
i don't know much about how to do css, but shouldn't this rule rather apply to only the select2 fields on that specific page? i mean if we generally do not want it to be 100%...